packages/validation-manager/src/ValidationManager.ts

```diff
@@ -237,7 +238,22 @@ export class ValidationManager implements IValidationManager {
       addresses: [],
       hash: ''
     }
-    const stateOverrideForEip7702 = await this.getAuthorizationsStateOverride(userOp.authorizationList ?? [])
+    const authorizationList = getAuthorizationList(userOp)
+    if (authorizationList.length > 0) {
+      // relevant only for RIP-7562...
+      requireCond(authorizationList.length === 1, 'Only one authorization is supported', ValidationErrors.InvalidFields)
+
+      const chainId = await this.provider.getNetwork().then(n => n.chainId)
+
+      // list is required to be of size=1. for completeness, we still scan it as a list.
```

Comment:
Why? we have ValidationManager7560, that will handle it as a list, why do we need it here?